timemachine/ff/handlers/nonbonded.py

```diff
@@ -349,7 +349,15 @@ def static_parameterize(params, smirks, mol):
 
 
 class SimpleChargeHandler(NonbondedHandler):
-    pass
+    def __init__(self, *args, **kwargs):
+        self.scale = 1
+        super().__init__(*args, **kwargs)
+
+    def partial_parameterize(self, params, mol):
+        return self.static_parameterize(params, self.smirks, mol) * self.scale
+
+    def parameterize(self, mol):
+        return self.static_parameterize(self.params, self.smirks, mol) * self.scale
```

Comment:
Document intent of `self.scale`? Introduces possibility for `static_parameterize` to have inconsistent behavior from `partial_parameterize` and `parameterize` (if `scale != 1`)

Reply:
will add documentation - note this is only defined on the `SimpleChargeHandler`, which we only ever use for production. This is to allow me to weaken the charges so that I can sample the "trans" conformation of `H-O-O-H` less frequently. I could just add another `smirnoff_1_1_0_rescale_sc.py` as an alternative?

Reply:
>note this is only defined on the SimpleChargeHandler, which we only ever use for production. This is to allow me to weaken the charges so that I can sample the "trans" conformation of H-O-O-H less frequently.

Okay, maybe a non-issue if not used for production, and if it simplifies some use that's needed in this PR.

(Currently I just don't understand the intent of putting it inside the handler in a way that makes `static_parameterize` inconsistent with `partial_parameterize`, instead of, say, explicitly multiplying the assigned charges by `scale` when needed (similar to how we rescale charges, etc. after parameterization in https://github.com/proteneer/timemachine/blob/d7c8ee5c1ab421ef17922621089ec910467bb9b8/timemachine/md/enhanced.py#L120-L124 and https://github.com/proteneer/timemachine/blob/776873a013e65f8d825570312d0fa87d3a438522/timemachine/fe/topology.py#L296-L309). Also note `scale != 1` will not preserve net charge if `net_charge != 0`.)

> I could just add another smirnoff_1_1_0_rescale_sc.py as an alternative?

This seems unnecessary

Reply:
The latter approach is fine too! but I'd need to subclass the `BaseTopology` class, or modify it to include a scale term. I can just make a `BaseTopologyRescaledCharges` as well.

Reply:
Can you just do `ff.q_handle.params *= 0.5`?

Reply:
That modifies it in a non-differentiable way (since it's not traceable in the constructor)

Reply:
actually sorry - I misinterpreted this! I thought you meant in the constructor - but it sounds like you just mean in the handle itself. To be clear, `ff.q_handle.params` are BCC parameters, so scaling the raw parameters only adjusts the polarization across bonds. However, we want the actual point charges to be scaled here.

Reply:
Makes sense